package/Distribution.xml

```diff
@@ -4,7 +4,7 @@
   <options customize="never" hostArchitectures="x86_64,arm64" rootVolumeOnly="true"/>
   <volume-check>
     <allowed-os-versions>
-        <os-version min="11.0.0"/>
+        <os-version min="12.0.0"/>
```

Comment:
Good catch! Not your mistake to fix but, if you're feeling generous, could you add a link to this file somewhere around here: https://github.com/Homebrew/brew/blob/d0eef12ea40ebb8e72b708336f8aa6d21d955061/Library/Homebrew/brew.sh#L507-L513

so this gets updated for future versions.

Reply:
I see you've done this already in https://github.com/Homebrew/brew/pull/16077 so going to keep the work there.